code/modules/admin/admin.dm

```diff
@@ -571,8 +572,16 @@ proc/admin_notice(var/message, var/rights)
 				<FONT COLOR='green'>Wanted issue for [src.admincaster_feed_message.author] successfully edited.</FONT><BR><BR>
 				<BR><A href='?src=\ref[src];ac_setScreen=[0]'>Return</A><BR>
 			"}
+		if(20) //comments but with more censorship!
+			dat+="<B>Comments:</B></BR>"
+			if(isemptylist(src.admincaster_viewing_message.comments))
+				dat+="No comments on this story yet!</BR>"
+			else
+				for(var/datum/feed_comment/COMMENT in src.admincaster_viewing_message.comments)
+					dat+="<BLOCKQUOTE style=\"padding:2px 4px;border-left:4px #797979 solid;\"><B>\[[world.time]\] [COMMENT.author]:</B>[COMMENT.message]<BR><A href='?src=\ref[src];ac_censorcomment=1;ac_comment=\ref[COMMENT]>Censor Comment</A></BLOCKQUOTE>"
+			dat+="<A href='?src=\ref[src];ac_setScreen=[9]'>Return</A>"
 		else
-			dat+="I'm sorry to break your immersion. This shit's bugged. Report this bug to Agouri, polyxenitopalidou@gmail.com"
+			dat+="Please report this on GitHub, along with what you did to make this appear."

```

Comment:
Since this is only going to be admin-facing and an OOC tool, and since I'm not familiar with the admin newscaster, so it may have unexpected issues, I'm going to keep this message.